packages/tasks/src/model-libraries.ts

```diff
@@ -446,6 +446,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pyannote_audio,
 		filter: true,
 	},
+	"py-feat": {
+		prettyLabel: "Py-Feat",
+		repoName: "py-feat",
```

Comment:
```suggestion
		repoName: "Py-Feat",
```

Reply:
Thanks @Wauplin this is all very helpful!

1. I updated all of my model cards to have a library and pipeline tag
2. set filter to false
3. removed py-feat from libraries-to-task.ts
4. I'll submit a new PR in the future if we ever have a more standardized approach to access models.
5. We have added `PyTorchModelHubMixin` mixin to all of our pytorch models and have played with the upload and download features. We ran into an issue using the upload method with safetensors, where the weights somehow got changed through that process and messed up inference.  I wasn't able to find other reports of this happening, nor did I investigate why. I instead just manually uploaded the `.pth` file to ensure everything worked properly.

This is the first step for us in integrating huggingface into py-feat. We will continue to refactor our codebase to streamline the integration as we continue to learn how everything works.

Thanks to everyone for creating such a great resource and being very helpful and responsive to new users!